src/views/Home.jsx

```diff
@@ -90,33 +93,58 @@ export function Home({ setListToken }) {
 	};
 
 	return (
-		<div className="Home">
-			<p>
-				Hello from the home (<code>/</code>) page!
+		<div className="home">
+			<p>Welcome to</p>
+			<h1>App Title!</h1>
+			<div className="app-logo">
+				<Image src={appImage} alt="app logo" />
+			</div>
+			<p className="app-summary">
+				App Title learns your buying habits and helps you remember what you're
+				likely to need to buy on your next shopping trip.
 			</p>
-			<Button onClick={createNewToken} variant="primary">
-				Create a new list
-			</Button>
-			<p>- or -</p>
-			<p>Join an existing shopping list by entering a three word token.</p>
-			<Form id="join-shopping-list-form" onSubmit={joinExistingToken}>
-				<Form.Group>
-					<Form.Label htmlFor="join-token">Share Token</Form.Label>
-					<Form.Control
-						type="text"
-						name="join-token"
-						id="join-token"
-						value={joinToken}
-						placeholder="three word token"
-						onChange={(e) => setJoinToken(e.target.value)}
-					/>
-				</Form.Group>
-				<Form.Group>
-					<Button type="submit" id="submit-share-token" variant="primary">
-						Join an existing list
-					</Button>
-				</Form.Group>
-			</Form>
+			<div className="text-center">
+				<Button onClick={createNewToken} className="btn-custom">
+					Create a new list
+				</Button>
+				<p>
+					<strong>- or -</strong>{' '}
+				</p>
+				<Form id="join-shopping-list-form" onSubmit={joinExistingToken}>
```

Comment:
Nit: This may be more of a stylistic preference, but I saw that bootstrap has a button addon that allows the submit button of an input to be connected to the Join List button.

If we'd like to remove this gap:
![image](https://user-images.githubusercontent.com/89554728/222385431-c96da0a2-84a8-4f83-8272-d819cd6991b2.png)

We can do something like this:
```suggestion
				<Form id="join-shopping-list-form" onSubmit={joinExistingToken}>
					<Form.Label htmlFor="join-token">List Token</Form.Label>
					<div className="d-flex flex-row justify-content-center">
						<InputGroup className="mb-3" style={{width: "300px"}}>
								<Form.Control
									type="text"
									name="join-token"
									id="join-token"
									value={joinToken}
									placeholder="three word token"
									onChange={(e) => setJoinToken(e.target.value)}
								/>
								<Button
									type="submit"
									id="submit-share-token"
									className="btn-custom"
								>
									Join List
								</Button>
						</InputGroup>
					</div>
				</Form>
```
And at the top we include the import of `InputGroup` from `react-bootstrap`. And of course the in-line css styling of `InputGroup` would be consolidated in into `Home.css`.

This will return something like this
![image](https://user-images.githubusercontent.com/89554728/222385079-09687344-24e3-4db9-8b1f-65508655d0ff.png)